tests/test_ic3.cpp

```diff
@@ -76,7 +76,12 @@ TEST_P(IC3UnitTests, SimpleSystemUnsafe)
   ASSERT_EQ(r, FALSE);
 }
 
-INSTANTIATE_TEST_SUITE_P(ParameterizedSolverIC3UnitTests,
-                         IC3UnitTests,
-                         testing::ValuesIn(available_solver_enums()));
+INSTANTIATE_TEST_SUITE_P(
+    ParameterizedSolverIC3UnitTests,
+    IC3UnitTests,
+    // TEMP excluding CVC4 because IC3 variants use two solvers
```

Comment:
Is this an issue in the tests only or are we also creating two instances of CVC4 anywhere in Pono?

Reply:
It could happen through the command line too, actually. I can add a check for that. If the user requests cvc4 with certain engines then we could end up creating two instances. Thanks for checking that!